app/Livewire/Public/DemoPage.php

```
@@ -0,0 +1,125 @@
+    {
+        return 'data';
+    }
+
+    public function render()
+    {
+        return view('livewire.public.demo-page');
+    }
+
+    public function submitDemo(): void
+    {
+        $data = $this->form->getState();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$data'. (Unused Code Rules)

(UnusedLocalVariable)